tools/mafft/mafft.xml

```diff
@@ -65,36 +80,73 @@
             <when value="mafft-linsi"/>
             <when value="mafft-qinsi"/>
             <when value="mafft-xinsi"/>
+            <when value="custom">
+                <param name="distance_method" type="select" display="radio" label="Distance method" help="Distance method must be chosen regarding your data">
+                    <option value="--6merpair" selected="true">Shared 6mers distance (fastest)</option>
+                    <option value="--globalpair">Global alignment (Needleman-Wunsch)</option>
+                    <option value="--localpair">Local alignment (Smith-Waterman)</option>
+                    <option value="--genafpair">Local, affine gap cost</option>
+                </param>
+                <param name="ep" type="float" value="0.123" label="Gap extend penalty" help="0.123 default value" />
```

Comment:
Regarding this label, the man page describe "ep" as "Offset value, which works like gap extension penalty, for group-to-group alignment", which seems more accurate. Also you can remove the default value from the `help` attribute.